hw_10/Square.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace hw_10
+{
+    class Square : IShape
+    {
+        public string Name { get; set; }
+        private int Side { get; set; }
```

Comment:
generally we do not using private properties. in this case you can use private field